athenahealth/appointment_status.go

```diff
@@ -0,0 +1,57 @@
+package athenahealth
+
+import "strconv"
+
+// AppointmentStatus is derived from https://docs.athenahealth.com/api/api-ref/appointment#Book-appointment
+// The athenaNet appointment status. There are several possible statuses.
+// x=cancelled
+// f=future (It can include appointments where were never checked in, even if the appointment date is in the past. It is up to a practice to cancel appointments as a no show when appropriate to do so.)
+// o=open
+// 2=checked in
+// 3=checked out
+// 4=charge entered (i.e. a past appointment)
+type AppointmentStatus string
+
+const (
+	AppointmentStatusCancelled     AppointmentStatus = "x"
+	AppointmentStatusChargeEntered AppointmentStatus = "4"
+	AppointmentStatusCheckedIn     AppointmentStatus = "2"
+	AppointmentStatusCheckedOut    AppointmentStatus = "3"
+	AppointmentStatusFuture        AppointmentStatus = "f"
+	AppointmentStatusOpen          AppointmentStatus = "o"
+)
+
+func (as AppointmentStatus) Int() int {
```

Comment:
Can you please share your use case for interacting with AppointmentStatus as an `int`? None of the existing values for appointment status in this repo reference it as an `int`.

Reply:
it was more for convenience I guess since they do send numbers back (in string format). i can remove this though no sweat.